spotiplex/plex.py

```diff
@@ -61,9 +81,18 @@ def find_playlist_by_name(self, playlist_name):
                 return playlist
         return None
 
+# Playlist is created in intervals of 300 since Plex's API will return a 414 URL TOO LONG inconsistently past that
     def create_playlist(self, playlist_name, playlist_id, tracks):
+        tracks_to_add = copy.deepcopy(tracks)
         try:
-            new_playlist = self.plex.createPlaylist(playlist_name, items=tracks)
+            iteration_tracks = tracks_to_add[:300]
+            del tracks_to_add[:300]
+            new_playlist = self.plex.createPlaylist(playlist_name, items=iteration_tracks)
+
+            while len(tracks_to_add) > 0:
+                iteration_tracks = tracks_to_add[:300]
+                del tracks_to_add[:300]
```

Comment:
If I'm understanding your question correctly, why we can't combine the `createPlaylist` call from above into this while loop? That's because the `createPlaylist` call requires there to be songs included in the call or else it will fail 🤔 
If you're referring to something else, I'm not following, sorry!